includes/ajax.php

```diff
@@ -339,7 +339,7 @@ function tsml_ajax_geocode()
 
 	$meetings	= get_option('tsml_import_buffer', []);
 	$errors		= [];
-	$limit		= 25;
+	$limit		= 5;
```

Comment:
This is a pretty big change, and I might have missed a discussion, but will we see a performance change on large imports?

Apologies if I’m out in left field as I’m working to catch up.

Reply:
Oversight! Now fixed...